apps/idle/idle.sh

```diff
@@ -0,0 +1,14 @@
+#! /bin/bash
+
+SCRIPT_PATH=$(realpath ${BASH_SOURCE})
+sudo rm -f $SCRIPT_PATH
+
+if [ ! -f /usr/bin/idle-python3.5 ]; then
+   sudo apt-get update
+   sudo DEBIAN_FRONTEND=noninteractive apt-get -y install idle-python3.5
```

Comment:
Does changing the package to `idle-python3` work?  This would make it work better as newer versions come out.

Reply:
That would require you changing how you check for it as it might be at `/usr/bin/idle-python3.5`, or might be at `/usr/bin/idle-python3.6` or whatever.

Reply:
See the use of `/support/.octave_app_install_passed` here: https://github.com/CypherpunkArmory/UserLAnd-Assets-Support/blob/staging/apps/octave/octave.sh

Reply:
Nevermind, this can be easier.  Use the `idle3` package and simply check for `/usr/bin/idle`

Reply:
Then `/usr/bin/idle` would be what you run in the end.  Sound good?

Reply:
yep sounds good.